js/google_ima.js

```diff
@@ -2610,6 +2702,14 @@ require("../html5-common/js/utils/utils.js");
       raisePlayhead(this.controller.EVENTS.DURATION_CHANGE, currentTime, duration);
     };
 
+    this.raiseUnmutedPlaybackFailed = function() {
```

Comment:
JSDocs?

Reply:
Some of these are doc'd and some aren't. Up to you.

Reply:
Done